azurelinuxagent/common/protocol/restapi.py

```diff
@@ -71,12 +71,6 @@ def __init__(self, family, version=None):
         self.version = "0.0.0.0" if version is None else version
         self.uris = []
 
-    def __eq__(self, other):
```

Comment:
This was moved to ExtensionsGoalState.compare, which allows me to add extra debug information to make it easier to follow up any differences in the goal states.